lib/csv_query/database.rb

```diff
@@ -1,37 +1,62 @@
 require 'sqlite3'
+require 'csv_query/table'
 
 module CsvQuery
-  # Wraps a SQLite in-memory database with a single table named csv.
+  # Wraps a SQLite in-memory database with a single table named csv by default if you pass in only a csv to initialize.
+  # Don't pass in a CSV and use #create_table and #import_data_from_csv to to perform more complex multi-table loads.
   class Database
-    attr_reader :database
+    attr_accessor :sqlite_database
```

Comment:
This is arguably personal preference.  When initializing this class, I tend to call it 'database'.  This results in 'database.database' to get to sqlite.  The real issue here is that ::Database should probably be called something else -- I just left its naming intact in fear of breaking backwards compatibility.  Maybe a better name would be #implementation, #sqlite, etc.